### PR TITLE
25 branch tong

### LINKED ISSUE #38: add batchnorm?

### PR DESCRIPTION
**Pull request by AmashiroNoah**
Resolve #38, test running test pass

### CODE REVIEW

File: project.ipynb [hunk 1]
```diff
@@ -367,12 +367,14 @@
     "    def __init__(self):\n",
     "        super(BaseCnn, self).__init__()\n",
     "        self.features = nn.Sequential(\n",
-    "            nn.Conv2d(3, 32, kernel_size=3, padding=1),  # Conv block 1\n",
+    "            nn.Conv2d(3, 32, kernel_size=3, padding=1),  # Conv block 1i\n",
```
Comment (TaillessSheep) [inline project.ipynb]: typo?
